src/client/mesh_generator_thread.cpp

```diff
@@ -231,6 +231,15 @@ void MeshUpdateQueue::cleanupCache()
 	g_profiler->avg("MeshUpdateQueue MapBlock cache size kB",
 			mapblock_kB * m_cache.size());
 
+	// Iterating the entire cache can get pretty expensive so don't do it too often
+	{
+		constexpr int cleanup_interval = 250;
```

Comment:
Where does 250ms come from? Seems fine, just wondering.

Reply:
don't want to delay cache purge too much (since that means stale entries stay around longer) but also shouldn't run too often
so 4 times a second it is.